imgui.go

```diff
@@ -638,3 +638,21 @@ func IsItemHoveredV(flags int) bool {
 func IsItemHovered() bool {
 	return IsItemHoveredV(HoveredFlagsDefault)
 }
+
+// ListBox creates a imgui::Listbox
```

Comment:
I know the imgui.h file doesn't provide any info on ListBox, though the comment can be more informative. Adding suggestion.
```suggestion
// ListBox creates a list of selectables of given items with equal height, enclosed with header and footer.
// The function returns true if the selection was changed. The value of currentItem will indicate the new selected item.
```